Doc/whatsnew/3.7.rst

```diff
@@ -97,6 +97,13 @@ New Modules
 Improved Modules
 ================
 
+http.server
+-----------
+:class:`~http.server.SimpleHTTPRequestHandler` supports the HTTP
+If-Modified-Since header. The server returns the 304 response status if the
+target file was not modified after the datetime specified in the header.
```

Comment:
Would it be better to just say “after the time specified in the header”? Datetime is just a Python implementation detail, not an English word or part of HTTP.

Also, I wonder if the details should go in the main documentation. Already <https://docs.python.org/dev/library/http.server.html#http.server.SimpleHTTPRequestHandler.do_GET> has quite a few details of the server behaviour.